lib/marin/src/marin/rl/rollout_worker.py

```diff
@@ -366,15 +346,7 @@ def _log_prompt_example(self, lesson_id: str, batch: RolloutBatch, step: int, ev
         logger.info(f"Eval sample for lesson {lesson_id} at step {step}: {metrics}")
 
     def _build_eval_metrics(self, prefix: str, lesson_id: str, batch: RolloutBatch) -> dict[str, Any]:
```

Comment:
probably can remove and just call the top-level? do we need this anymore or was it only used by the hooks